src/components/modules/paste.ts

```diff
@@ -306,31 +313,66 @@ export default class Paste extends Module {
     }
   }
 
+  /**
+   * Get tags from tags configuration.
+   *
+   * @param {string | object} tagsConfig - tag name or sanitize config object.
+   * @returns {string[]} array of tags.
+   */
+  private getTags = (tagsConfig: (string | object)): string[] => {
```

Comment:
seems like this method could return `undefined` as well in case if `tagConfig` will be passed as something different than `string` or `object`. 

Could you handle this please? Because for now the Exception will occur in the `[...tags]` statement

Reply:
Still actial

Reply:
return empty array